tests/on_target/tests/test_uart_output.py

```diff
@@ -65,3 +65,20 @@ def test_program_board_and_check_uart(t91x_board, hex_file):
     patterns_button_press.append(re.compile(r"location_module: location_event_handler: Got location: lat: ([\d.]+), lon: ([\d.]+), acc: ([\d.]+)"))
 
     check_uart_data(t91x_board.uart, patterns_button_press)
+
+    t91x_board.uart.flush()
+    # Simulate network disconnect
+    t91x_board.uart.write("lte offline\r\n")
+
+    time.sleep(5)
+
+    t91x_board.uart.write("lte normal\r\n")
+
+    patterns_lte_reconnect = patterns_button_press.copy()
+    patterns_lte_reconnect.insert(0, re.compile(r""))
+    patterns_lte_reconnect.append(re.compile(r"network: Network connectivity lost"))
+    patterns_lte_reconnect.append(re.compile(r"trigger: disconnected_exit: disconnected_exit"))
```

Comment:
should this be:
`re.compile(r"trigger: disconnected_entry: disconnected_entry"),`